root/components/list/AreaList.js

```diff
@@ -19,32 +22,51 @@ import {
 } from '../../utility/tableColumns';
 
 type Props = {
+  ...CollectionCommentsRoleT,
   +areas: $ReadOnlyArray<AreaT>,
   +checkboxes?: string,
   +mergeForm?: MergeFormT,
   +order?: string,
+  +showCollectionComments?: boolean,
   +sortable?: boolean,
 };
 
 const AreaList = ({
   areas,
+  canEditCollectionComments,
   checkboxes,
+  collectionComments,
+  collectionId,
   mergeForm,
   order,
+  showCollectionComments = false,
```

Comment:
Is there any case where we'd be passing `collectionComments` and want this to be false?  Seems like we can just show them if `collectionComments` is non-empty.

Reply:
IIRC my worry was that this should still work if no entities in the collection have a comment yet.